README.md

```diff
@@ -1,6 +1,6 @@
 # Impact Client
 
-Impact is an advanced Minecraft Utility Mod aimed towards user friendliness.
+Impact is an advanced Minecraft Utility Mod aimed towards user-friendliness.
```

Comment:
no

Reply:
the correct way 😎

Reply:
<img width="749" alt="Screen Shot 2020-12-05 at 12 57 12 PM" src="https://user-images.githubusercontent.com/3837873/101264461-6853a180-36f9-11eb-9804-ace1f4d83e50.png">

Reply:
trolled